### DISCUSSION TITLE
Didn't work with docker swarm

### DISCUSSION THREAD

**Discussion by zorion79** [umputun/reproxy#65] [Ideas]
If my service run on one node and reproxy on another node then reproxy didn't find service container. If both run on one node then all working.
API for services exist. https://docs.docker.com/engine/api/v1.41/#operation/ServiceList
Can we add this function?

**Comment by umputun** [comment]
well, it exists with some form of orchestration on top of the docker, in this case - the swarm. Docker itself can be accessed remotely but such configuration is unusual and even exotic to be supported here.

The closest thing to such a multi-node discovery will be the support of consul catalog #56 as soon as @negasus completes it. 

_I moved it to discussions because this is outside of the core functionality of the reproxy_
